core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1476,6 +1480,24 @@ object KafkaConfig {
       .define(RaftConfig.QUORUM_RETRY_BACKOFF_MS_CONFIG, INT, Defaults.QuorumRetryBackoffMs, null, LOW, RaftConfig.QUORUM_RETRY_BACKOFF_MS_DOC)
   }
 
+  object RequestMetricsSizeBucketsValidator extends Validator {
```

Comment:
To avoid code duplication between this validator and the one in getRequestSizeMetricNameMap,
maybe we should run the validation when converting from the string value to a list of ints inside KafkaConfig,
similar to 
https://github.com/linkedin/kafka/blob/efe6c6bffef97d3198ab73ea92ec922c20d27ca7/core/src/main/scala/kafka/server/KafkaConfig.scala#L1948

Reply:
updated